packages/colors-block/src/components/list/ListItem.tsx

```diff
@@ -96,8 +103,19 @@ export const ListItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onDe
                 {colorSpaces?.map((colorSpaceId) => {
                     const mappedColorSpace = mapColorSpaces(colorSpaceId, color);
 
+                    if (!isEditing && !mappedColorSpace.value) {
+                        return <></>;
```

Comment:
do you need to return <></> or could you just `return;`?

Reply:
Need to return something from a `.map`